src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -943,12 +943,25 @@ default Set<ExceptionalDeath> deathSafeExceptions() {
         return EnumSet.of(ExceptionalDeath.FIGHT_CAVE, ExceptionalDeath.INFERNO);
     }
 
+    @ConfigItem(
+        keyName = "deathIgnoredRegions",
+        name = "Ignored Region IDs",
+        description = "User-specified, comma-separated Region IDs where deaths should be ignored.<br/>" +
+            "Use the '::dinkregion' chat command or an online map to obtain the region IDs.<br/>" +
+            "For example, Prifddinas spans 12894, 12895, 13150, 13151",
+        position = 45,
+        section = deathSection
+    )
+    default String deathIgnoredRegions() {
+        return "4904, 5930, 6953, 7723";
```

Comment:
perhaps we use some obviously fake IDs so its clearer that users can delete them

```suggestion
        return "123, 456, 789";
```